src/lib/UIKitConfigProvider/utils/types.ts

```diff
@@ -0,0 +1,13 @@
+// @link https://gist.github.com/navix/6c25c15e0a2d3cd0e5bce999e0086fc9
+/* eslint-disable @typescript-eslint/ban-types */
+export type DeepPartial<T> = T extends Function
```

Comment:
Can we move this to `typeHelpers`

Reply:
Ah I should've mention about it too. I'll move the whole logic under `lib/UIKitConfigProvider` to https://github.com/sendbird/sendbird-uikit-core-ts/tree/main/packages/uikit-tools so RN and React can share the logic. What do you think?

Reply:
Cool